src/fairseq2/recipes/wav2vec2/asr/common.py

```diff
@@ -64,11 +64,9 @@ def update_ctc_loss(self, batch: Seq2SeqBatch, loss: Tensor) -> None:
         :param ctc_loss:
             The loss of ``batch``.
         """
-        batch_size = torch.tensor(batch.batch_size)
+        normalized_loss = loss / batch.batch_size / math.log(2)
 
-        normalized_loss = loss.cpu() / batch_size / math.log(2)
```

Comment:
Why was it `loss.cpu()` before?

Reply:
Just for a bit of consistency. `batch_size` was a CPU tensor, so wanted them to be on the same device.